specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2017-12-01-preview/mysql.json

```diff
@@ -0,0 +1,1725 @@
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource ID"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "TrackedResource": {
+      "description": "Resource properties including location and tags for track resources.",
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "The location the resource resides in."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Application-specific metadata in the form of key-value pairs."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "required": [
+        "location"
+      ]
+    },
+    "ServerProperties": {
+      "properties": {
+        "administratorLogin": {
+          "type": "string",
+          "description": "The administrator's login name of a server. Can only be specified when the server is being created (and is required for creation)."
+        },
+        "version": {
+          "$ref": "#/definitions/ServerVersion",
+          "description": "Server version."
+        },
+        "sslEnforcement": {
+          "$ref": "#/definitions/SslEnforcement",
+          "description": "Enable ssl enforcement or not when connect to server."
+        },
+        "userVisibleState": {
+          "type": "string",
+          "description": "A state of a server that is visible to user.",
+          "enum": [
+            "Ready",
+            "Dropping",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "ServerState",
+            "modelAsString": true
+          }
+        },
+        "fullyQualifiedDomainName": {
+          "type": "string",
+          "description": "The fully qualified domain name of a server."
+        },
+        "earliestRestoreDate": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Earliest restore point creation time (ISO8601 format)"
+        },
+        "storageProfile": {
+          "$ref": "#/definitions/StorageProfile",
+          "description": "Storage profile of a server."
+        }
+      },
+      "description": "The properties of a server."
+    },
+    "StorageProfile": {
+      "properties": {
+        "backupRetentionDays": {
+          "type": "integer",
+          "description": "Backup retention days for the server."
+        },
+        "geoRedundantBackup": {
+          "type": "string",
+          "description": "The tier of the particular SKU, e.g. Basic.",
```

Comment:
Please update the description

Reply:
That's already done in my local branch. i'll push it.